design/resources.go

```diff
@@ -122,6 +122,28 @@ var _ = a.Resource("workitem.2", func() {
 		a.Response(d.InternalServerError)
 	})
 
+	a.Action("update", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.PATCH("/:id"),
+		)
+		a.Description("update the given work item with given id.")
```

Comment:
thanks for correcting, done in https://github.com/almighty/almighty-core/pull/482/commits/d8c08aea9e5f6cc075bed42bc55fb0972ec34018